vtex/styleguide
# Adds LocationIcon

#### What is the purpose of this pull request?

Adds a LocationIcon needed for `PlaceComponents`.

#### What problem is this solving?

The `DeviceCoordinates` component (from `vtex-apps/place-components`) requires these icons, and instead of defining them there, we decided to share them in the styleguide since they might be needed by other projects too.

#### How should this be manually tested?

In your VTEX IO app, try to import the newly added icon with `import { IconLocation } from 'vtex.styleguide'`, and then use it like: `<IconLocation />` or `<IconLocation granted />`. Test that you can modify its size and color like with most other icons.

[Workspace](https://styleguidelocation--checkoutio.myvtex.com/places). The `DeviceCoordinates` component should be displaying the icon.

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/25920622/74563159-ed8f7f00-4f4a-11ea-8439-cbcaa2fa8e26.png)

#### Types of changes
